app/(gcforms)/[locale]/(app administration)/admin/(with nav)/accounts/[id]/manage-permissions/actions.ts

```diff
@@ -1,25 +1,26 @@
 "use server";
+
 import { updatePrivilegesForUser } from "@lib/privileges";
-import { checkPrivilegesAsBoolean } from "@lib/privileges";
 import { revalidatePath } from "next/cache";
 import { authCheckAndThrow } from "@lib/actions";
 
+// Public facing functions - they can be used by anyone who finds the associated server action identifer
+
 export const updatePrivileges = async (
   userID: string,
   privilegeID: string,
   action: "add" | "remove"
 ) => {
   const { ability } = await authCheckAndThrow();
-  if (checkPrivilegesAsBoolean(ability, [{ action: "update", subject: "User" }])) {
```

Comment:
Is removing the `checkPrivilegesAsBoolean` part of 

"Removes duplication of privilege checks for some server actions"

Reply:
Yes it is.